Src/java/tools/cql-formatter/src/test/resources/org/cqframework/cql/tools/formatter/git-issue-437.cql

```diff
@@ -0,0 +1,6 @@
+define "Encounter With Principal Procedure of SCIP VTE Selected Surgery":
```

Comment:
Please add a comment to the beginning of the library with a link to the github issue and a short description explaining the bug and the resolution